app/models/organisation.rb

```diff
@@ -46,7 +47,9 @@ class Organisation < ApplicationRecord
       .or(where(id: SchoolGroupMembership.select(:school_id).where(school_group_id: registered_organisations)))
   end)
 
-  scope :visible_to_jobseekers, -> { schools.not_closed.or(Organisation.trusts).registered_for_service }
+  scope :not_out_of_scope, -> { where.not(detailed_school_type: Organisation::OUT_OF_SCOPE_DETAILED_SCHOOL_TYPES).or(where(detailed_school_type: nil)) }
+
```

Comment:
I had to add in `or(where(detailed_school_type: nil)` because I found that rails where not queries do not return instances that have nil values in the specified field which is pretty strange. Can check it out here if you are interested: https://thoughtbot.com/blog/activerecord-s-where-not-and-nil